### ISSUE TITLE
Minor issue with drop shadow at bottom of login form

### ISSUE THREAD

**Issue by bfackland** [ploneintranet/ploneintranet.prototype#101]
It appears that on first load (or hard-refresh) the last ~15% of the login form popup has no drop-shadow:

![screen shot 2015-06-18 at 18 17 30](https://cloud.githubusercontent.com/assets/2538526/8396196/2397f098-1d94-11e5-8374-1b7e3b4a7a63.png)

**Comment by cornae** [comment]
Yes. And each time you load the form again it might look different. I think it's because the measuring of the height of the modal doesn't look at images being loaded already or not. So it's a race condition issue perhaps. @jcbrand We discussed the modal measuring regarding to layout modifiers, but not sure if we covered images as well. Did we? Should this ticket go on your name for this or does it fit into the other issues with modals and layout modifiers?

**Comment by jcbrand** [comment]
@cornae's hunch sounds likely. You can assign it to me and I'll take a look.
